src/mson.js

```diff
@@ -0,0 +1,238 @@
+/*
+ * Renders refract elements into MSON.
+ */
+
+/*
+ * Indent a piece of multiline text by a number of spaces.
+ * Setting `first` to `true` will also indent the first line.
+ */
+function indent(input, spaces, options = {first: false}) {
```

Comment:
Isn't https://github.com/apiaryio/fury-adapter-apib-serializer/pull/1/files#diff-e52a9a67956bd51d044af4058fb939ceR16 just a repeat of this?